### DISCUSSION TITLE
GHES system resource alerts

### DISCUSSION THREAD

**Discussion by TristanLee9608** [community/community#15299] [Enterprise] [answered]
Hi, GitHub Developers

I know that we could configure collectd to send system resource data to outside programs such as Splunk, Grafana... But GitHub Enterprise already has Grafana in Monitoring tab in Management console, and also smtp to send emails and such. 

Could we have a page for setting system resource threshold and send alerts when cpu, memory,etc.. resource goes over that threshold using smtp or Grafana Alerts?

**Comment by djdefi** [comment, marked as answer]
As you mentioned [External monitoring](https://docs.github.com/en/enterprise-server/admin/enterprise-management/monitoring-your-appliance/setting-up-external-monitoring) via `collectd`, SNMP, or [log forwarding](https://docs.github.com/enterprise-server/admin/monitoring-activity-in-your-enterprise/exploring-user-activity/log-forwarding) are the current options for alerting. 

Thresholds and alerts are not configurable in GitHub Enterprise's [Monitor Dashboard](https://docs.github.com/enterprise-server/admin/enterprise-management/monitoring-your-appliance/accessing-the-monitor-dashboard) now, but I agree that having the ability to configure resource alerting would be super useful for administrators.
